internal/app/exporter/exporter.go

```diff
@@ -407,6 +409,28 @@ func (c *rdsCollector) getQuotasMetrics(client servicequotas.ServiceQuotasClient
 	c.logger.Debug("quota metrics fetched", "metrics", metrics)
 }
 
+func (c *rdsCollector) getInstanceTagLabels(dbidentifier string, instance rds.RdsInstanceMetrics) (keys []string, values []string) {
+	labels := map[string]string{
+		"aws_account_id": c.awsAccountID,
+		"aws_region":     c.awsRegion,
+		"dbidentifier":   dbidentifier,
+	}
+
+	// Add instance tags to labels
+	// Prefix label containing instance's tags with "tag_" prefix to avoid conflict with other labels
+	for k, v := range instance.Tags {
+		labelName := fmt.Sprintf("tag_%s", k)
+		labels[labelName] = v
+	}
+
+	for k, v := range labels {
+		keys = append(keys, k)
+		values = append(values, v)
+	}
+
+	return
```

Comment:
(non-blocking) In newer version of go, map will probably be moved as a standard package but if you want to go ahead you could just do

return maps.Keys(labels), maps.Values(labels)

Reply:
I tested this during development, but keys and values are returned in an indeterminate order, which results in incorrect values for labels:

> Values returns the values of the map m. The values will be in an indeterminate order.
> https://pkg.go.dev/golang.org/x/exp/maps#Values